src/main/java/edu/harvard/iq/dataverse/util/DateUtil.java

```diff
@@ -0,0 +1,67 @@
+package edu.harvard.iq.dataverse.util;
+
+import edu.harvard.iq.dataverse.DataverseLocaleBean;
+
+import java.sql.Timestamp;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+/**
+ *
+ * @author jchengan
+ */
+public class DateUtil {
+
+    public static String formatDate(Date dateToformat) {
+        String formattedDate;
+        DateFormat dateFormatter;
+        try{
+            DataverseLocaleBean d = new DataverseLocaleBean();
+            Locale currentLocale = new Locale(d.getLocaleCode());
+
+            dateFormatter = DateFormat.getDateInstance(DateFormat.DEFAULT, currentLocale);
+            formattedDate = dateFormatter.format(dateToformat);
+
+            return formattedDate;
+        }
+        catch(Exception e) {
+            return null;
+        }
+    }
+
+    public static String formatDate(String dateToformat){
+        String formattedDate = "";
+        DateFormat inputFormat = DateFormat.getDateInstance(DateFormat.DEFAULT);
```

Comment:
@JayanthyChengan You changed the input format from hardcoded to the date instance dafault? Any particular reason? since this is how we store the date, it's fine to use the hardcoded way which matches how we store, I think. (it feels more readable to me). I can be convinced, though.